dashboard/src/components/jobs/parameters/TimeParameters.vue

```diff
@@ -27,40 +30,33 @@
     </div>
     <div class="timefield">
       <b>Start:</b>
-      <datetime
-        @close="emitParams"
-        placeholder="Click here to set start date"
-        type="datetime"
-        v-model="start"
-        format="y-LL-dd HH:mmZZ"
-        :zone="timezone"
-        :picker-zone="timezone"
-        :max-datetime="end"
-      ></datetime>
+      <datetimefield @update-datetime="setStart" :timezone="timezone" />
+      <help helpText="The value of the first timestamp in your data." />
     </div>
     <div class="timefield">
       <b>End:</b>
-      <datetime
-        @close="emitParams"
-        type="datetime"
-        placeholder="Click here to set start date"
-        v-model="end"
-        :zone="timezone"
-        :picker-zone="timezone"
-        :min-datetime="start"
-        format="y-LL-dd HH:mmZZ"
-      ></datetime>
+      <datetimefield @update-datetime="setEnd" :timezone="timezone" />
+      <help
+        helpText="The last minute covered by your data. For example, 60 minute data with a last timestamp at 2020-12-31 23:00 should have an end of 2020-12-31 23:59."
```

Comment:
This is actually a little more flexible. In this example anything after 23:00 would work and 2021-01-01 00:00 would also work. Not sure of a good way to describe it though